README.md

```diff
@@ -12,106 +12,139 @@
-- Keep it SIMPLE - it's not nearly as complicated as it first may look.
-- You must [submit your challenge](https://airtable.com/shrUGm2T8TYCFAmjN) by the deadline, wherever you get to.
-- Use your own test framework and evidence your test-driven development by committing on passing tests.
-- Please write your own README detailing how to install your project, how to run the tests, how you approached the problem and provide screenshots of interacting with your program.
-- If you refer to the solution of another coach or student, please put a link to that in your README.
-- Please create separate files for every class, module, and spec.
-
-## Steps
+## Installation Steps
 
 1. Fork this repo, and clone to your local machine
 2. `npm install` to install project dependencies
-3. Convert stories into a representative domain model and test-drive your work.
-4. Run your tests using `npm test` or `node specRunner.js`
-5. OPTIONAL: [Lint](https://eslint.org/docs/user-guide/getting-started) your source code using `npx eslint src`.
+3. Run the tests using `npm test` or `node specRunner.js`
 
 ## Task
 
+#### Convert stories into a representative domain model and test-drive your work.
+
 We have a request from a client to write the software to control the flow of planes at an airport. The planes can land and take off provided that the weather is sunny. Occasionally it may be stormy, in which case no planes can land or take off. Here are the user stories that we worked out in collaboration with the client:
 
-#### Acceptance Criteria
+#### User story - 1
 
 ```
 As an air traffic controller
 So I can get passengers to a destination
 I want to instruct the airport to land a plane
+```
 
-Initial thoughts:
-1. Need an Air Traffic Controller.
-2. Air traffic controller need to be able to instruct Airport.
-3. Airport should be able to land a plane.
+**Initial thoughts:**
 
-**Test 1** - `basketItems` length increases when `addItem` is called with *instance of `Item`*
+1. Need an Airport, that has an array of planes in that airport.
+2. Need a plane.
+3. `landPlanes` should only take in instances of _Plane_
+4. Airport should be able to land a plane, increasing the number of planes in airport.
 
-## Domain Models
+_Test 1_ - `planesInAirport[` length increases when `landPlane()` is called.\
+_Test 2_ - Test `landPlane()` by passing in a string.\
+_Test 3_ - Test `landPlane()` by passing in a falsy value, `null`, `undefined`.
 
-| Objects               | Properties                        | Messages            | Outputs |
-| --------------------  | -------------------------         | -----------------   | ------- |
-| Airport               |   planesInAirport @Array[@Plane]  |  landPlane(@Plane)  |  @Void  |
+#### Domain Model - 1
 
+| Objects | Properties                     | Messages          | Outputs |
+| ------- | ------------------------------ | ----------------- | ------- |
+| Airport | planesInAirport @Array[@Plane] | landPlane(@Plane) | @Void   |
+
```

Comment:
Should've added Plane as an object here to clarify, a plane is being passed to when landPlane is called.